.circleci/config.yml

```diff
@@ -778,6 +790,44 @@ jobs:
               echo "Not building or pushing to dockerhub for tag=${CIRCLE_TAG} branch=${CIRCLE_BRANCH}"
             fi
 
+  deploy-postgres-python-utils:
+    docker:
+      - image: docker:stable-git
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    steps:
+      - setup_remote_docker
+      - checkout
+      - display-versions
+      - write-version
+      - run:
+          name: Build and deploy to Dockerhub
+          command: |
+            export UTILS_DOCKERHUB_REPO=mozilla/sync-postgres-py-utils
```

Comment:
This Docker Hub repo does not exist at the moment.  I've filed a support ticket to have it created.